docs/manual/index.html

```diff
@@ -1063,66 +1063,145 @@ <h3 id="map_calls">Avoiding calls to specific methods</h3>
 </p>
 
 <p>
-Alternately, you can provide a different implementation of a method that will be used
-when running under Randoop.  Randoop transforms the code under test,
-replacing each call to a undesired method by a call to a new implementation
-that you provide.
+Alternately, you can use the <code>mapcall</code> agent to provide a different
+implementation of a method that will be used when running under Randoop.
+The mapcall Java agent transforms the code under test, replacing each call to a
+undesired method by a call to a new implementation.
+Default alternate implementations are provided for <code>System.exit()</code>
```

Comment:
Please avoid passive voice.  It isn't clear what a user needs to do to achieve this effect.